fuzz/cli.rs

```diff
@@ -176,8 +238,14 @@ fn run_honggfuzz(target: &str) -> Result<(), Error> {
     let mut rust_flags = env::var("RUSTFLAGS").unwrap_or_default();
     rust_flags.push_str("-Z sanitizer=address");
 
-    let mut hfuzz_args = env::var("HFUZZ_RUN_ARGS").unwrap_or_default();
-    hfuzz_args.push_str("--exit_upon_crash --run_time 5");
+    let hfuzz_args = format!(
+        "-f {} \
+         --exit_upon_crash \
+         --run_time 5 \
```

Comment:
This is a mistake introduced by me. Looks like "run_time 5" is not needed at all. You can help change it (or leave to me XD)